### PR TITLE
second part of filesystem tests conversion to kotlin

### PR DESCRIPTION
**Pull request by brianPlummer**

### CODE REVIEW

**Comment by brianPlummer** [comment]
@digitalbuddha @pavlospt

File: filesystem/src/test/java/com/nytimes/android/external/fs3/UtilTest.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        val child = mock(File::class.java)
+        val parent = mock(File::class.java)
+        `when`(child.canonicalFile).thenReturn(child)
+        `when`(child.parentFile).thenReturn(parent)
+        `when`(parent.isDirectory).thenReturn(true)
+        util.createParentDirs(child)
+        verify(parent).mkdirs()
+    }
+
+    companion object {
```
Comment (pavlospt) [inline filesystem/src/test/java/com/nytimes/android/external/fs3/UtilTest.kt]: Seems to be unused!

**Review by pavlospt — APPROVED** [review]

File: filesystem/src/test/java/com/nytimes/android/external/fs3/UtilTest.kt (hunk 1 above)
Comment (brianPlummer) [reply filesystem/src/test/java/com/nytimes/android/external/fs3/UtilTest.kt]: done!

File: filesystem/src/test/java/com/nytimes/android/external/fs3/SourceDiskDaoStoreTest.kt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+class SourceDiskDaoStoreTest {
+    @Mock
+    lateinit var fetcher: Fetcher<BufferedSource, BarCode>
+    @Mock
+    lateinit var diskDAO: SourcePersister
+    private val barCode = BarCode("value", KEY)
+
+    @Test
+    fun testSimple() {
```
Comment (wdziemia) [inline filesystem/src/test/java/com/nytimes/android/external/fs3/SourceDiskDaoStoreTest.kt]: Nit: Can we rename this to something more descriptive?

File: filesystem/src/test/java/com/nytimes/android/external/fs3/SourceFilerReaderWriterStoreTest.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    @Mock
+    lateinit var fetcher: Fetcher<BufferedSource, BarCode>
+    @Mock
+    lateinit var fileReader: SourceFileReader
+    @Mock
+    lateinit var fileWriter: SourceFileWriter
+    private val barCode = BarCode("value", KEY)
+
+    @Test
+    fun testSimple() {
```
Comment (wdziemia) [inline filesystem/src/test/java/com/nytimes/android/external/fs3/SourceFilerReaderWriterStoreTest.kt]: Nit: Can we rename this to something more descriptive?

**Review by wdziemia — APPROVED** [review]

File: filesystem/src/test/java/com/nytimes/android/external/fs3/SourceDiskDaoStoreTest.kt (hunk 2 above)
Comment (brianPlummer) [reply filesystem/src/test/java/com/nytimes/android/external/fs3/SourceDiskDaoStoreTest.kt]: done!

File: filesystem/src/test/java/com/nytimes/android/external/fs3/SourceFilerReaderWriterStoreTest.kt (hunk 3 above)
Comment (brianPlummer) [reply filesystem/src/test/java/com/nytimes/android/external/fs3/SourceFilerReaderWriterStoreTest.kt]: done!

**Pull request merged**